src/app/services/export.service.ts

```diff
@@ -167,7 +167,8 @@ export class ExportService {
       this.resourceService.runExportJob(
         job,
         this.exportDataTypes,
-        this.variableService.combination,
+        this.constraintService.cohortSelectionConstraint,
+        this.constraintService.variableConstraint(this.variableService.variables),
```

Comment:
this would produce a combination constraint with all the variables, is that intended?

Reply:
what happens to the previous approach where variableService.combination is used?